pkgs/dart_services/pubspec.yaml

```diff
@@ -28,6 +30,7 @@ dev_dependencies:
   build_runner: ^2.4.5
   dart_flutter_team_lints: ^2.0.0
   grinder: ^0.9.4
+  json_serializable: any
```

Comment:
If there was a separate package for the JSON protocol it would be more clear where I need to run `build_runner` to regenerate the serialization code.

Reply:
+1 - I'll refactor this to use a shared `pkgs/dartpad_shared` or similar package.

Reply:
That would be amazing!